source/fundamentals/serialization/poco.txt

```diff
@@ -207,22 +255,337 @@ Omit Empty Fields
+.. code-block:: csharp
+   :copyable: true
+   :emphasize-lines: 5
+
+   public class PatientRecord
+   {
+       public Guid Id { get; set; }
+
+       [BsonDateTimeOptions(DateOnly = true)]
+       public DateTime DateOfBirth { get; set; }
+   }
+
+You can also use the ``[BsonDateTimeOptions()]`` attribute to specify the ``DateTimeKind``
+of a ``DateTime`` property. In the following code example, the ``PatientRecord`` class
+has an ``AppointmentTime`` property of type ``DateTime``.
+The ``[BsonDateTimeOptions(Kind = DateTimeKind.Local)]`` attribute indicates
+that the time component of the property's value is in local time. When the
+driver serializes this property, it converts the time to UTC, the standard
+format for times stored in MongoDB.
+
+.. code-block:: csharp
+   :copyable: true
+   :emphasize-lines: 5
+
+   public class PatientRecord
+   {
+       public Guid Id { get; set; }
+
+       [BsonDateTimeOptions(Kind = DateTimeKind.Local)]
+       public DateTime AppointmentTime { get; set; }
+   }
+
+You can also specify one or both of the previous ``DateTime`` options when
+registering the class map:
+
+.. code-block:: csharp
+   :copyable: true
+
+   BsonClassMap.RegisterClassMap<House>(classMap => 
+   {
+      classMap.AutoMap();
+      classMap.MapMember(p => p.DateOfBirth)
+         .SetSerializer(new DateTimeSerializer(dateOnly: true));
+      classMap.MapMember(p => p.AppointmentTime)
+         .SetSerializer(new DateTimeSerializer(DateTimeKind.Local));
+   });
+
+.. tip:: DateTimeKind Values
+
+   The ``DateTimeKind`` enum is part of the {+framework+}. For more information on
+   its members, see the `Microsoft documentation for the DateTimeKind enum. <https://learn.microsoft.com/en-us/dotnet/api/system.datetimekind?view=net-8.0>`__
+
+Customize Dictionary Serialization
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``DictionaryRepresentation`` enum defines the formats the {+driver-short+}
+can serialize a ``Dictionary`` instance to. This enum includes the following members:
+
+- **Document**: (Default) The driver serializes the ``Dictionary`` to a ``BsonDocument``.
+  Each entry in the dictionary is a ``BsonElement`` with a name equal to
+  the entry's key and a value equal to the entry's value. You can
+  use this representation only when all the keys in the dictionary are strings that
+  are also valid ``BsonElement`` names.
+- **ArrayOfArrays**: The driver serializes the dictionary to a ``BsonArray``. Each
+  entry in the dictionary is a nested, two-element ``BsonArray`` that contains
+  the entry's key and the entry's value.
+- **ArrayOfDocuments**: The driver serializes the dictionary to a ``BsonArray``. Each
+  entry in the dictionary is a nested ``BsonDocument`` of the form
+  ``{ k : key, v : value }``. Because the keys and values are
+  tagged with element names, you can query this format more intuitively than
+  an ``ArrayOfArrays``.
+
+In the following code example, the ``RoomSize`` property is a dictionary that contains
```

Comment:
S: to better match the description as this property likely describes more than one room - applies to all mentions
```suggestion
In the following code example, the ``RoomSizes`` property is a dictionary that contains
```